### ISSUE TITLE
TOSCA 2.0 support

### ISSUE THREAD

**Issue by cdanger** [tliron/turandot#22]
Do you have any plan to support TOSCA 2.0 in the near future?

**Comment by tliron** [comment]
Since Turandot uses Puccini, any dialect Puccini supports works in Turandot. I'll leave this issue open to track progress.

**Comment by cdanger** [comment]
Makes sense. So I asked a [similar question on the Puccini github](https://github.com/tliron/puccini/discussions/143).
